### DISCUSSION TITLE
Set hard heap limit

### DISCUSSION THREAD

**Discussion by AnupNadahalli** [xerial/sqlite-jdbc#788] [Q&A]
How to set hard heap limit mentioned in the below available pragmas?
https://www.sqlite.org/pragma.html#pragma_hard_heap_limit

**Comment by gotson** [comment]
New pragmas must be added in the code to be handled.

**Reply by AnupNadahalli** [reply]
Not sure when it was added but are you saying that as of now it is not supported ?

**Reply by gotson** [reply]
It's not supported, the list of known pragmas are here : https://github.com/xerial/sqlite-jdbc/blob/8c78a665c3ee2375ce90ed3262e7b94681b73cf0/src/main/java/org/sqlite/SQLiteConfig.java#L376
